tests/components/weatherkit/__init__.py

```diff
@@ -0,0 +1,70 @@
+"""Tests for the Apple WeatherKit integration."""
+from unittest.mock import patch
+
+from apple_weatherkit import DataSetType
+
+from homeassistant.components.weatherkit.const import (
+    CONF_KEY_ID,
+    CONF_KEY_PEM,
+    CONF_SERVICE_ID,
+    CONF_TEAM_ID,
+    DOMAIN,
+)
+from homeassistant.const import CONF_LATITUDE, CONF_LONGITUDE
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry, load_json_object_fixture
+
+
+async def init_integration(
+    hass: HomeAssistant,
+    is_night_time: bool = False,
+    has_hourly_forecast: bool = True,
+    has_daily_forecast: bool = True,
+) -> MockConfigEntry:
+    """Set up the WeatherKit integration in Home Assistant."""
+    entry = MockConfigEntry(
+        domain=DOMAIN,
+        title="Home",
+        unique_id="0123456",
+        data={
+            CONF_LATITUDE: 35.4690101707532,
+            CONF_LONGITUDE: 135.74817234593166,
+            CONF_KEY_ID: "QABCDEFG123",
+            CONF_SERVICE_ID: "io.home-assistant.testing",
+            CONF_TEAM_ID: "ABCD123456",
+            CONF_KEY_PEM: "-----BEGIN PRIVATE KEY-----\nwhateverkey\n-----END PRIVATE KEY-----",
+        },
+        pref_disable_polling=True,
```

Comment:
Why do we disable polling?

Reply:
I think I was worried about making excessive API calls, but that doesn't make sense because we're mocking the responses. So I guess it's fine to remove this line.